htdocs/core/tpl/card_presend.tpl.php

```diff
@@ -383,6 +383,12 @@
 	$formmail->param['returnurl'] = $_SERVER["PHP_SELF"].'?id='.$object->id;
 	$formmail->param['fileinit'] = array($file);
 
+	$parameters = array();
+	$reshook = $hookmanager->executeHooks('FromEmail', $parameters, $formmail);
```

Comment:
It seems you already have the getFormMail hook inside the 
$formmail->get_form();
called 2 lines after.